src/pages/components/modal/code.mdx

```diff
@@ -1,34 +1,43 @@
 ---
 title: Modal
 description: Modals focus the user’s attention exclusively on one task or piece of information via a window that sits on top of the page content.
-tabs: ['Code', 'Usage', 'Style', 'Accessibility']
+tabs: ["Code", "Usage", "Style", "Accessibility"]
 ---
 
 <ComponentDemo
```

Comment:
Is there anyway to add more content to the "hasScrollingContnet" modal so it can actually scroll?